gulpfile.js

```diff
@@ -266,13 +266,12 @@ gulp.task('react:modules', function() {
 });
 
 gulp.task('react:extract-errors', function() {
-  return merge(
-    gulp.src(paths.react.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactDOM.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactNative.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactTestRenderer.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactNoopRenderer.src).pipe(extractErrors(errorCodeOpts))
```

Comment:
@gaearon I rebased and removed `reactNoopRenderer` from the list of `extract-errors` targets. Please let me know if it makes sense!